### ISSUE TITLE
Option to cancel download options

### ISSUE THREAD

**Issue by kokyriakidis** [KarchinLab/open-cravat#86]
Hi!

I tried to generate an excel file for some files and it is spinning for several days

![image](https://user-images.githubusercontent.com/33731293/144709754-c2555e24-3de8-4a5e-bc5e-39e9a56f0add.png)

I do not want this excel file and there is no such option to cancel my request. For example it would be nice to click again on "Excel" button to cancel it while it is generated.

**Comment by rkimoakbioinformatics** [comment]
Hi @kokyriakidis Thanks for letting us know. We'll look into it. Can you let me know the job ID of the problem job? I can investigate what happened with the Excel report generation if you want.

**Comment by kokyriakidis** [comment]
211126-204806 and 211126-204715

**Comment by rkimoakbioinformatics** [comment]
@kokyriakidis I have generated the excel result files for the two jobs. Please check if they are as expected. Generating a result file can take a while and if the server stops responding and thus is restarted in the middle of the result file generation, what you observed can happen, for example. In one of the next versions we'll consider adding "cancel report generation" feature. Until then, if you notice similar behaviors please let us know and we'll handle them manually.

**Comment by kokyriakidis** [comment]
Thank you very much for your support!

**Comment by kokyriakidis** [comment]
@rkimoakbioinformatics Excel has the following limit 

```
Total number of rows and columns on a worksheet | 1,048,576 rows by 16,384 columns
```
So, any file with more than 1,048,576 variants (including the header line) will have missing rows (variants). I guess you should first check for this number before allowing any user to create an excel file.

**Comment by rkimoakbioinformatics** [comment]
@kokyriakidis Thanks for the information. Indeed that makes sense. I have published excelreporter v2.1.1 which will skip report generation if the number of unique variants is greater than 1048576.
